src/goose/brokers/redis/batch.clj

```diff
@@ -0,0 +1,25 @@
+(ns goose.brokers.redis.batch
+  (:require [goose.brokers.redis.commands :as redis-cmds]
+            [goose.defaults :as d]
+            [taoensso.carmine :as car]))
+
+(defn- set-batch-state
+  [{:keys [id callback-fn-sym jobs]}]
+  (let [batch (d/prefix-batch id)
```

Comment:
rename var to batch-state-key or something?

Reply:
Renamed it to `batch-state-key`